airflow-core/src/airflow/api_fastapi/core_api/routes/public/auth.py

```diff
@@ -55,3 +55,23 @@ def logout(request: Request, next: None | str = None) -> RedirectResponse:
         logout_url = request.app.state.auth_manager.get_url_login()
 
     return RedirectResponse(logout_url)
+
+
+@auth_router.get(
+    "/refresh",
+    responses=create_openapi_http_exception_doc([status.HTTP_307_TEMPORARY_REDIRECT]),
+)
+def refresh(request: Request, next: None | str = None) -> RedirectResponse:
+    """Refresh the authentication token."""
+    refresh_url = request.app.state.auth_manager.get_url_refresh()
+
+    if not refresh_url:
+        return RedirectResponse(request.app.state.auth_manager.get_url_logout())
```

Comment:
There is a route in core Airflow for logout, you should redirect to this one. It uses underneath the `get_url_logout` form auth manager

Reply:
Sure, I got the base url and redirected directly to /auth/logout endpoint and updated the tests accordingly in test_auth